recipes/tmap/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "tmap" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/reymond-group/tmap/archive/9aa043dd1af022161048ee0a4b5554ec5e34977f.zip
+  sha256: 56cf4f83d663950ab7ef41deb425b8e1c01de378fb5239aeef7b6613a6edf212
+
+  patches:
+    - 01_bind2d_deprecated.diff
+    - 02_setup_windows.diff
+
+build:
+  number: 0
+  skip: True  # [win]
+  binary_relocation: False  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python {{ python }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - setuptools
+    - pybind11
+    - ogdf ==2020.02
+  host:
+    - vc  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - python
+    - pip
+  run:
+    - vc  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
```

Comment:
I don't know what the status of `openmp` on Windows it, so I don't know what to suggest for Windows.
For Linux/maxOS it suffices to have them in `build` because of "strong" `run_exports`, ref http://conda-forge.org/docs/maintainer/knowledge_base.html#openmp.

Reply:
Oh, apparently the  `openmp` package is not updated anymore since https://github.com/conda-forge/openmp-feedstock/pull/25.
So, either you probably need `llvm-openmp` on Windows too, or some other package (or it's provided via compiler? IDK).

@isuruf: Do you know what is recommended on Windows regarding OpenMP for packages on conda-forge?

Reply:
tmap is not built on windows at the moment. We can address this in the repo later.